src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -489,4 +486,16 @@
   <data name="Show_completion_list_after_a_character_is_deleted" xml:space="preserve">
     <value>Show completion list after a character is _deleted</value>
   </data>
+  <data name="Always_indent_switch_case" xml:space="preserve">
```

Comment:
Tagging @jinujoseph.

I think modifying the resource names may impact localization.